BuildForSDG/team-010-frontend
# setup frontend ci-cd pipelines

#### What does this PR do?
- modifies webpack output
- configures build process for deployment on heroku
- configures containerized test on TravisCI
- Configures Test Coverage on coveralls.io

#### Description of Task to be completed?
#### How should this be manually tested?
- Check heroku frontend staging - https://staged-team-010-frontend.herokuapp.com/
- check travisCI build - https://travis-ci.com/github/BuildForSDG/team-010-frontend/builds/166576292
- check coveralls coverage - https://coveralls.io/builds/30840562

#### Any background context you want to provide?
- Please take note that all test env variable would henceforth be added on .travis.yml file
- please Ignore the currently unkown badges on the branch, the badges can only reflect the develop branch which would begin after merge

#### What are the relevant pivotal tracker stories?


#### Screenshots (if appropriate)
#### Questions:
